cli/src/semgrep/formatter/sarif.py

```diff
@@ -30,17 +53,22 @@ def _create_sarif_location_dict(
         nesting_level: int = -1,
     ) -> Mapping[str, Any]:
         message = f"{var_type}: '{snippet.strip()}' @ '{str(location.path.value)}:{str(location.start.line)}'"
+        (
+            extended_snippet,
+            start_highlight,
+            end_highlight,
+        ) = SarifFormatter._generate_taint_snippet(location)
         sarif_dict = {
             "location": {
                 "message": {"text": message},
                 "physicalLocation": {
                     "artifactLocation": {"uri": str(rule_match.path)},
                     "region": {
-                        "startLine": location.start.line,
+                        "startLine": start_highlight,
```

Comment:
`startLine`??
or you meant `startColumn`?

Reply:
you're correct

Reply:
done